src/test/resources/inputs/basics/datetime.sql

```diff
@@ -12,37 +12,40 @@ SELECT TIME '01:02:03' FROM T;
 --#[datetime-03]
 SELECT TIME '01:02:03.456' FROM T;
 
---#[datetime-04]
-SELECT TIME '01:02:03.456-00:00' FROM T;
+-- --#[datetime-04]
+-- previous test includes timezone offset for TIME
+-- SELECT TIME '01:02:03.456-00:00' FROM T;
 
 --#[datetime-05]
-SELECT TIME '01:02:03.456+00:00' FROM T;
+SELECT TIME WITH TIME ZONE '01:02:03.456+00:00' FROM T;
 
 --#[datetime-06]
-SELECT TIME '01:02:03.456+00:30' FROM T;
+SELECT TIME WITH TIME ZONE '01:02:03.456+00:30' FROM T;
 
 --#[datetime-07]
-SELECT TIME '01:02:03.456-00:30' FROM T;
```

Comment:
self-review: some time and timestamp tests should have used datetime literals with the `WITH TIME ZONE` keywords. also since we don't support Ion's timestamp directly, commented-out the negative `0` timezone offset test, `datetime-04`